src/app/shared/theme-picker/theme-picker.html

```diff
@@ -1,18 +1,15 @@
-<button mat-icon-button [mat-menu-trigger-for]="themeMenu" matTooltip="Select a theme!"
-        tabindex="-1">
+<button mat-icon-button [mat-menu-trigger-for]="themeMenu" aria-label="Select a theme"
+        matTooltip="Select a theme for the documentation">
   <mat-icon>format_color_fill</mat-icon>
 </button>
 
-<!-- TODO: replace use of `mat-menu` here with a custom overlay -->
-<mat-menu class="docs-theme-picker-menu" #themeMenu="matMenu" x-position="before">
-  <mat-grid-list [cols]="2">
-    <mat-grid-tile *ngFor="let theme of themes">
-      <div mat-menu-item (click)="installTheme(theme.name)">
-        <div class="docs-theme-picker-swatch">
-          <mat-icon class="docs-theme-chosen-icon" *ngIf="currentTheme === theme">check_circle</mat-icon>
-          <div class="docs-theme-picker-primary" [style.background]="theme.primary"></div>
-        </div>
-      </div>
-    </mat-grid-tile>
-  </mat-grid-list>
+<mat-menu #themeMenu="matMenu" xPosition="before" class="theme-picker-menu">
+  <button mat-menu-item *ngFor="let theme of themes" (click)="onThemeSelection(theme.name)">
```

Comment:
This button also needs an `aria-label`

Reply:
The screen reader announces the content of the button which is in the `<span>` on line 12. I.e. "Deep Purple slash Amber, menu item 1 of 4". That seems sufficient?

Reply:
I missed that span